moveit_planners/chomp/chomp_motion_planner/src/chomp_optimizer.cpp

```diff
@@ -204,7 +204,11 @@ void ChompOptimizer::initialize()
   for (size_t i = 0; i < joint_model_group_->getFixedJointModels().size(); i++)
   {
     const moveit::core::JointModel* model = joint_model_group_->getFixedJointModels()[i];
-    fixed_link_resolution_map[model->getName()] = model->getParentLinkModel()->getParentJointModel()->getName();
+    // In case this is the base joint (eg: a fixed virtual_joint) then we need to check for NULL
+    if (model->getParentLinkModel())
```

Comment:
I would prefer a 
```
if (!model->getParentLinkModel())
  continue;
```
However, that's probably a matter of taste.

Reply:
I think it's better to avoid goto-derivatives in general.

Reply:
As I said, that's a matter of taste. Leave it like it is.

Reply:
+1 to using ``continue`` or ``return`` early